client/src/app/App.tsx

```diff
@@ -55,6 +57,10 @@ class App extends Component<RouteComponentProps<AppRouteProps>, AppState> {
      */
     private readonly _networkService: NetworkService;
 
+    private readonly configId = process.env.REACT_APP_CONFIG_ID ?? "local";
```

Comment:
Don't load the config again just pass it in as a property of the `App` component from `Index`